charts/ccd-case-management-web/values.preview.template.yaml

```diff
@@ -6,19 +6,20 @@ nodejs:
   ingressHost: ${SERVICE_FQDN}
   environment:
     IDAM_LOGIN_URL: https://idam-web-public.aat.platform.hmcts.net
-    CCD_GW_LOGOUT_URL: https://ccd-api-gateway-web-pr-273.service.core-compute-preview.internal/logout
-    CCD_API_URL: https://ccd-api-gateway-web-pr-273.service.core-compute-preview.internal/aggregated
-    CCD_DATA_URL: https://ccd-api-gateway-web-pr-273.service.core-compute-preview.internal/data
-    CCD_GW_OAUTH2_URL: https://ccd-api-gateway-web-pr-273.service.core-compute-preview.internal/oauth2
-    DM_URL: https://ccd-api-gateway-web-pr-273.service.core-compute-preview.internal/documents
+    CCD_GW_LOGOUT_URL: https://gateway-ccd.aat.platform.hmcts.net/logout
+    CCD_API_URL: https://gateway-ccd.aat.platform.hmcts.net/aggregated
+    CCD_DATA_URL: https://gateway-ccd.aat.platform.hmcts.net/data
+    CCD_GW_OAUTH2_URL: https://gateway-ccd.aat.platform.hmcts.net/oauth2
+    DM_URL: https://gateway-ccd.aat.platform.hmcts.net/documents
     DM_URL_REMOTE: ^https?://(?:api-gateway\.preprod\.dm\.reform\.hmcts\.net|dm-store-aat\.service\.core-compute-aat\.internal(?::\d+)?)/documents
-    ANNOTATION_API_URL: https://ccd-api-gateway-web-pr-273.service.core-compute-preview.internal/em-anno
-    PRINT_SERVICE_URL: https://ccd-api-gateway-web-pr-273.service.core-compute-preview.internal/print
+    ANNOTATION_API_URL: https://gateway-ccd.aat.platform.hmcts.net/em-anno
+    PRINT_SERVICE_URL: https://gateway-ccd.aat.platform.hmcts.net/print
     PRINT_SERVICE_URL_REMOTE: https://return-case-doc-ccd.nonprod.platform.hmcts.net
-    CCD_ACTIVITY_URL: https://ccd-api-gateway-web-pr-273.service.core-compute-preview.internal/activity
-    PAYMENTS_URL: https://ccd-api-gateway-web-pr-273.service.core-compute-preview.internal/payments
+    CCD_ACTIVITY_URL: https://gateway-ccd.aat.platform.hmcts.net/activity
+    PAYMENTS_URL: https://gateway-ccd.aat.platform.hmcts.net/payments
+    SHUTTER_REDIRECT_URL: https://xui-mo-webapp.aat.platform.hmcts.net
```

Comment:
We should have both url and wait time in values.yaml, and overwrite it for envs if needed. It looks like we don't have them in values.yaml

Reply:
Addressed 👍